tomcat/src/main/java/org/apache/coyote/http11/HtmlLoader.java

```diff
@@ -10,14 +10,15 @@
 public class HtmlLoader {
 
     private static final String DEFAULT_HTML = "Hello world!";
+    private static final String DELIMITER = System.lineSeparator();
 
     public static String generateFile(String fileName) throws IOException {
         URL resource = HtmlLoader.class.getClassLoader().getResource(fileName);
         if (resource == null) {
             return DEFAULT_HTML;
         }
-        final Path path = new File(resource.getFile()).toPath();
-        final List<String> actual = Files.readAllLines(path);
-        return String.join("\n", actual) + "\n";
+        Path path = new File(resource.getFile()).toPath();
+        List<String> actual = Files.readAllLines(path);
+        return String.join(DELIMITER, actual) + DELIMITER;
```

Comment:
```suggestion
        return actual.stream()
                .collect(Collectors.joining(DELIMITER, "", DELIMITER));
```
이렇게 하면 suffix를 join에 함께 넣어서 문자열로 만들 수 있을 것 같아요 :)

Reply:
수정하였습니다